data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+			`	Partway there, a hooded figure suddenly falls in step beside you.`
+			choice
+				`	(Keep walking.)`
+				`	(Ask them who they are.)`
+			`	"Sssshhhh!" says the voice at your elbow, "do not react, and do not be alarmed, it is merely I."`
+			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise, it is Sayari.`
+			`	"I should not be coming to you with this, so you must promise to say nothing of it to anyone just yet," she says.`
+			choice
+				`	"Of course."`
+			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
+			`	"Of course, a thorough investigation has occurred on our side, and no evidence of such things has turned up, but..."`
+			`	"Well, you see we cannot investigate your space openly or easily. I have paid it little mind, written it off as one of her eccentricities, but things are very tense right now. The Council of Elders wants to maintain our secrecy, the human diplomats are willing to accommodate us, but we sense their governments grow tired of maintaining the fiction. We are well aware that human intelligence services are highly competent, and motivated by many different factors. It worries me that one of your factions may not be wholly honest, and may have knowledge they intend to use as ammunition if things don't work out well for them."`
+			`	You have not had much exposure to this elected "Council of Elders", but it seems they must hold some kind of final authority over decisions.`
+			choice
+				`	"But what should I do about it?"`
+			`	"You are well-known, and must be canny to have got where you are. A Hai in trouble might recognize you, or you might perceive them when others would not. It would be in the interests of both our people to be sure that there is no such issue. Would you look for me? For both our sakes?"`
+			branch "about that"
+				"hai slave prereq" >= 1
+			choice
+				`	"Sure, I can have a look around."`
+					accept	
+			label "about that"
+			`	"Well, about that," you say, "I think I might have already come across some evidence of this before."`
+			`	Sayari's hood nearly slips off as she looks up at you with a horrified expression. "Surely not?!"`
+			`	You rush to reassure her. "Nothing concrete, of course. I can go and look to be sure."`
+				accept
+	to complete
+		"hai slave prereq" >= 3
+	on complete
+		log `You found proof of more kidnapped Hai. Sayari's fears appear to have been entirely justified...`
+
+mission "Hai Leaks Response 7"
+	landing
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
+		not attributes "uninhabited"
+	to offer
+		has "Hai Leaks Response 6: active"
+		"hai slave prereq" >= 3
```

Comment:
Why are the missions that increment this in the next file? I think that either those missions should be moved into this file, or these missions that rely on this condition should be moved into that file, just so that we don't need to jump back and forth between files when trying to read through this.

Reply:
Because they're under Nomadic's copyright, the main reasons the files break up where they do is to do with who did what.

Reply:
I'd rather not split content based on trying to get it under the copyright header that it matches. That's not something we've ever concerned ourselves with in the past. Content should go in the file that makes sense for where it is in the story, and the PR, through either the commit history or through mention in the description, should mention who did what.

Reply:
Mmm, I can see that. There is a legitimate conceptual divide between them though. You separate out from a structured A-B-C narrative to a "Go collect these in any order" narrative, and then back into an A-B-C narrative in the next file.

The file in question is also something I'd like to expand on in the future and it's much neater in its own file for that than tacked onto the end of the previous one.